### PR TITLE
change ResultSet->createPaginatedQuery parameter ORM\AbstractQuery

### PR DESCRIPTION
**Pull request by EaredSeal**

### CODE REVIEW

File: src/Kdyby/Doctrine/ResultSet.php [hunk 1]
```diff
@@ -360,10 +360,10 @@ public function count()
 
 
 	/**
-	 * @param ORM\Query $query
+	 * @param ORM\AbstractQuery $query
 	 * @return ResultPaginator
 	 */
-	private function createPaginatedQuery(ORM\Query $query)
+	private function createPaginatedQuery(ORM\AbstractQuery $query)
```
Comment (fprochazka) [inline src/Kdyby/Doctrine/ResultSet.php]: this wouldn't match the signature https://github.com/doctrine/doctrine2/blob/master/lib/Doctrine/ORM/Tools/Pagination/Paginator.php#L60

Comment (EaredSeal) [inline src/Kdyby/Doctrine/ResultSet.php]: Without this adjustment can not use NativeQueryBuilder
https://github.com/Kdyby/Doctrine/blob/master/src/Kdyby/Doctrine/NativeQueryBuilder.php#L76

Comment (fprochazka) [inline src/Kdyby/Doctrine/ResultSet.php]: You can, you just have to define the count SQL yourself.

**Comment by fprochazka** [comment]
Closing this, as the paginator cannot (to my knowledge) generate a count query for native query.

**Pull request closed without merging**